slf4j-jdk-platform-logging/src/test/java/org/slf4j/jdk/SLF4JSystemLoggerTest.java

```diff
@@ -0,0 +1,77 @@
+package org.slf4j.jdk;
+
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.io.ByteArrayOutputStream;
+import java.io.PrintStream;
+import java.util.ArrayList;
+import java.util.ServiceLoader;
+
+import static org.junit.Assert.*;
+
+public class SLF4JSystemLoggerTest {
+
+    private static final PrintStream ERROR_OUT = System.err;
+    private static final ByteArrayOutputStream OUTPUT = new ByteArrayOutputStream();
+    private static final PrintStream ERROR_OUT_REPLACEMENT = new PrintStream(OUTPUT);
+
+    @Before
+    public void setUp() {
+        System.setErr(ERROR_OUT_REPLACEMENT);
+    }
+
+    @After
+    public void tearDown() {
+        System.setErr(ERROR_OUT);
+    }
+
+    @Test
+    public void loggerFinderLoadedAsOnlyService() {
+        // this method asserts that there is exactly one `LoggerFinder` and its of the correct type
```

Comment:
Is this an actual requirement? It could be over-specified. We only would have to change the `getSLF4JSystemLoggerFinder` method to filter the `SLF4JSystemLoggerFinder` (see below).

Reply:
Yeah, that makes sense.